lib/meetup.ts

```diff
@@ -0,0 +1,67 @@
+export async function getEvents() {
+  const gql = `
+query {
+  groupByUrlname(urlname: "devICT") {
+    upcomingEvents(input:{first: 10, reverse: true}) {
+      edges {
+        node {
+          title
+          dateTime
+          shortUrl
+        }
+      }
+    }
+  }
+}
+`.trim();
+
+  const respData = await fetch("https://api.meetup.com/gql", {
+    method: "POST",
+    body: JSON.stringify({ query: gql }),
+  }).then((r) => r.json());
+
+  const events = respData.data.groupByUrlname.upcomingEvents.edges.map((e) => ({
+    ...e.node,
+    date: new Date(e.node.dateTime),
+  }));
+
+  const eventsThisWeek = events.filter((e) => {
+    const now = new Date();
+    const endOfNextWeek = new Date(now.getTime() + 7 * 24 * 60 * 60 * 1000);
+    return e.date > now && e.date < endOfNextWeek;
+  });
+
+  const days = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  const weekday = (date) =>
+    date.toLocaleString("en-us", {
```

Comment:
These can be typed now!
```suggestion
  const weekday = (date: Date) =>
```